src/main/java/frc/robot/commands/NFRResetWristCommand.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.commands;
+
+import org.northernforce.subsystems.arm.NFRRotatingArmJoint;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class NFRResetWristCommand extends Command {
+    NFRRotatingArmJoint m_wrist;
+    public NFRResetWristCommand(NFRRotatingArmJoint wrist)
+    {
+        m_wrist = wrist;
+    }
+    @Override public void initialize()
+    {
+        addRequirements(m_wrist);
+        m_wrist.getController().getAbsoluteEncoder().get().setAbsoluteOffset(Rotation2d.fromDegrees(25).getRotations());
```

Comment:
This might not work as expected, as this will simply put the offset to a constant, which may be incorrect as the encoder will not always be aligned perfectly. This should be `MathUtil.inputModulus(Rotation2d.fromDegrees(25).getRotations() - wrist.getController().getAbsoluteEncoder().get().getAbsolutePosition() + wrist.getController().getAbsoluteEncoder().get().getAbsoluteOffset(), 0, 1)`